openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -185,15 +186,31 @@ def _verify_response(self, access_token, expected_status_code, expected_cookie_n
         if expected_cookie_name:
             self.assertIn(expected_cookie_name, response.cookies)
 
-    def test_success(self):
+    def _create_dot_access_token(self, grant_type='Client credentials'):
+        """
+        Create dot based access token
+        """
+        dot_application = dot_factories.ApplicationFactory(user=self.user, authorization_grant_type=grant_type)
+        return dot_factories.AccessTokenFactory(user=self.user, application=dot_application)
+
+    def test_dop_access_token(self):
         access_token = AccessToken.objects.create(
```

Comment:
Nit: For readability and consistency, consider refactoring this test so there's an analogous `_create_dop_access_token` helper method.